API/api/services/event_service.py

```
@@ -8,18 +8,13 @@ class EventService:
-            raise ValidationError(serializer.errors)
-
+    def create_event(self, validated_data: dict) -> dict:
         try:
             event = self.repo.create(
-                name=serializer.validated_data["name"],
-                event_date=serializer.validated_data["event_date"],
+                name=validated_data["name"],
+                event_date=validated_data["event_date"],
             )
         except IntegrityError:
             raise ValidationError({"event_date": "Un événement existe déjà pour cette date."})
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)